ui/tests/integration/components/replication-dashboard-test.js

```diff
@@ -0,0 +1,67 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const REPLICATION_DETAILS = {
+  state: 'stream-wals',
+  primaryClusterAddr: 'https://127.0.0.1:8201',
+};
+
+const REPLICATION_DETAILS_SYNCING = {
+  state: 'merkle-diff',
+  primaryClusterAddr: 'https://127.0.0.1:8201',
+};
+
+module('Integration | Enterprise | Component | replication-dashboard', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('replicationDetails', REPLICATION_DETAILS);
+    this.set('replicationDetailsSyncing', REPLICATION_DETAILS_SYNCING);
+    this.set('componentToRender', 'replication-secondary-card');
+    this.set('clusterMode', 'secondary');
+    this.set('isSecondary', true);
+  });
+
+  test('it renders', async function(assert) {
+    await render(hbs`<ReplicationDashboard 
+    @replicationDetails={{replicationDetails}} 
+    />`);
+
+    assert.dom('[data-test-replication-dashboard]').exists();
+  });
+
+  test('it renders table rows', async function(assert) {
+    await render(hbs`<ReplicationDashboard @replicationDetails={{replicationDetails}}/>`);
+    assert.dom('[data-test-table-rows').exists();
+  });
+
+  test('it renders with primary cluster address when set, and documentation link', async function(assert) {
+    await render(hbs`<ReplicationDashboard 
+    @replicationDetails={{replicationDetails}} 
+    @clusterMode={{clusterMode}}
+    @isSecondary={{isSecondary}}
+    />`);
+
+    assert
+      .dom('[data-test-primary-cluster-address]')
```

Comment:
this assertion + the assertion on line 37 can be moved into the first test, and then you can just change the assertion message. i.e. `assert.dom('[data-test-table-rows').exists('renders table rows')`

Reply:
oh, i'd also add an assertion that there's not an alert banner by default too. a good thing to keep in mind when writing tests is 1) what do i _not_ want to show by default and 2) what do i _want_ to show by default, and adding assertions for both.

Reply:
Good call!  I'll amend